extdata/package_list.csv

```diff
@@ -19,7 +19,7 @@
 "knitr","A General-Purpose Package for Dynamic Report Generation in R [@R-knitr]","1.14"
 "lubridate","Make Dealing with Dates a Little Easier [@R-lubridate]","1.6.0"
 "microbenchmark","Accurate Timing Functions [@R-microbenchmark]","1.4.2.1"
-"profvis","Interactive Visualizations for Profiling R Code [@R-profvis]","0.3.2.9000"
+"profvis","Interactive Visualizations for Profiling R Code [@R-profvis]","0.3.2"
```

Comment:
Where should I add this for travis though? Only see install packages code that's not evaluated on Travis workers...